src/Views/inputView.js

```diff
@@ -0,0 +1,14 @@
+import { Console } from '@woowacourse/mission-utils';
+
+const getCarNamesInput = async () => {
+  const carNames = await Console.readLineAsync(
+    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분): ',
+  );
+  return carNames;
+};
+
+const getRacingCountInput = async () => {
+  const racingCount = await Console.readLineAsync('시도할 횟수는 몇 회인가요?: ');
+  return racingCount;
+};
```

Comment:
복잡하지 않은 로직이라면 return을 통해 바로 반환하는 방법도 고려해보면 좋을 것 같아요.

Reply:
세광님 안녕하세요 :) 
리뷰 감사드립니다! 

@Potalzz 
궁금한 부분은 그렇게 하였을 때, 가독성 측면에서도 더욱 좋아지는걸까요?

Reply:
저도 세광님 말처럼 아래 코드 처럼 바로 return 해도 된다고 생각해요. 사실 이미 함수명에서 getRacingCountInput() 이라고 쓰여 있어서 변수에 racingCount라고 표현하지 않아도 racingCount 값을 입력받아서 그 값을 return 하는 구나를 알 수 있거든요. 함수명과 변수명으로 두 번을 알려주지 않아도 된다고 생각합니다☺️
```suggestion
const getRacingCountInput = () => {
  return Console.readLineAsync('시도할 횟수는 몇 회인가요?: ');
};
```